.github/prod-cluster/frontend-service.yaml

```diff
@@ -14,12 +14,15 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: frontend
+  name: frontend-clusterip
```

Comment:
Good question, I should have documented this better in the description, I just updated the last bullet point to reflect this. Typically, this renaming allows a safe deployment for:
1. This specific deployment process, `kubectl apply -f .github/prod-cluster/frontend-service.yaml` without conflict with existing `frontend` service already in the cluster, because the `type` field is immutable.
2. Regardless this PR, the service by default as `LoadBalancer` (in `~/kubernetes-manifests`) has the same name, so historically deploying this `NodePort` service should have already complained (immutable type field). In OnlineBoutique, the default one is named `service-external`.
Hope that makes sense, with that said, I don't see any issue around that, let me know if you have in mind soemething else.